pkg/operation/botanist/component/kubeapiserverexposure/kube_apiserver_sni.go

```diff
@@ -186,6 +192,32 @@ func (s *sni) Deploy(ctx context.Context) error {
 		return err
 	}
 
+	if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, s.client, accessControlAPIServer, func() error {
+		accessControlAPIServer.Labels = getLabels()
+		spec, err := s.getAccessControlAPIServerSpec()
+		if err != nil {
+			return err
+		}
+
+		accessControlAPIServer.Spec = *spec.DeepCopy()
```

Comment:
Do we need the `DeepCopy()` here? New structures are anyways getting generated every time `getAccessControlAPIServerSpec` is called and they aren't being reused anywhere else.

Reply:
If we do not DeepCopy here we get a linter error because the Spec got a Mutex Lock Object that get assigned / copied. Thats the only way to pass linting :)

Reply:
Thanks for the explanation! :)